kimchi/src/constants.rs

```diff
@@ -0,0 +1,5 @@
+/// The number of points that are evaluated in the protocol (zeta and zeta * omega).
```

Comment:
this file needs a module doc comment (`//!`)

Reply:
Add module doc in https://github.com/o1-labs/proof-systems/pull/974/commits/b2bb15aa2556a627152b1b800e7ebc5cb239a682
Pls check if that is what you want, or similar one. @mimoo